src/components/marker/timeline-marker.tsx

```diff
@@ -0,0 +1,48 @@
+import {h} from 'preact';
+import {ui} from '@playkit-js/kaltura-player-js';
+import * as styles from './timeline-marker.scss';
+import type {TimelineMarkerProps} from '../../types/timelineTypes';
+import {A11yWrapper} from '@playkit-js/common';
```

Comment:
to reduce bundle size use 
`import {A11yWrapper, OnClick} from '@playkit-js/common/dist/hoc/a11y-wrapper';`